heliopy/data/test/test_data.py

```diff
@@ -53,12 +54,16 @@ def setup_class(self):
     def test_mag(self):
         df = cassini.mag_hires(self.starttime, self.endtime)
         check_datetime_index(df)
+        if type(df) is sunpy.timeseries.timeseriesbase.GenericTimeSeries:

```

Comment:
Instead of adding this if check in all the functions, it would be easier to add it to the `check_units` function. Maybe you could raise a warning there if the data isn't a `TimeSeries`, so we can see in the test output what still needs to be converted.

Reply:
Oh yes, that sounds way better!